cirq-google/cirq_google/json_test_data/GateTabulation.json

```diff
@@ -1,519 +0,0 @@
-{
```

Comment:
We should have moved the `GateTabulation.json` to `GateTabulation.json_inward`; which matches the repr in `GateTabulation.repr_inward` ? I see the `GateTabulation.repr_inward` file but I don't see the `GateTabulation.json_inward` file.

Reply:
Shoot thanks for the catch. It's in my local repo but wasn't added.

Reply:
Actually what happened was that `GateTabulation.json_inward` was already there and unmodified. Same with `GateTabulation.repr_inward`, but the class name was modified to be `cirq.TwoQubitGateTabulation`. So we are all good on that front.